src/compas/data/encoders.py

```diff
@@ -182,14 +182,14 @@ class DataDecoder(json.JSONDecoder):
 
     >>> import json
     >>> from compas.data import DataDecoder
-    >>> with open('point.json', 'r') as f:
-    ...     point = json.load(f, cls=DataDecoder)
+    >>> with open('point.json', 'r') as f:        # doctest: +SKIP
+    ...     point = json.load(f, cls=DataDecoder) # doctest: +SKIP
```

Comment:
just out of curiosity why these needs to be skipped? So in doctest the read and writing to disk is not possible? If that's the case perhaps these kind of stuff can be `loads` and `dumps` instead